eng/scripts/Update-Snippets.ps1

```diff
@@ -19,8 +18,22 @@ if ($ServiceDirectory -and ($ServiceDirectory -ne "*")) {
 
 if (-not (Test-Path env:TF_BUILD)) { $StrictMode = $true }
 
-if($StrictMode) {
-    Resolve-Path "$root" | %{ dotnet run -p $generatorProject -b "$_" -sm -c Release }
-} else {
-    Resolve-Path "$root" | %{ dotnet run -p $generatorProject -b "$_" -c Release }
+nuget install SnippetGenerator `
```

Comment:
This is going to fail on non-windows. Don't you want to use `dotnet tool install`?